### ISSUE TITLE
Automatic expansion of certain paths

### ISSUE THREAD

**Issue by jalanmiddleton** [DeveloperLiberationFront/Excel-Function-Visualizer#26]
Should there be a way to make the tree automatically expand some of the following?
- Longest path (indicating deepest parse depth/possible most complex)
- Leaf node with highest frequency (most common pattern of all?)

**Comment by Gmaster89** [comment]
The toggle for Longest path and Leaf Node can be added in treeMaker.js and when clicked, expansion can be handled in "function expandclick()"

**Comment by jalanmiddleton** [comment]
Do you have any idea of how these might be integrated into the interface smoothly? Like, would they be new buttons somewhere, or some new key combination?

**Comment by Gmaster89** [comment]
Yes, this can be integrated as an option to pop up for the right click on parent node (and if at all there is a need to find a longest path from a particular nth node in the tree, then this left click functionality can be applied for the child nodes also).
So, apart from the parent node, we are also able to find the longest path from a particular node at nth level too.
